crates/core_simd/src/lane_count.rs

```diff
@@ -31,13 +33,13 @@ macro_rules! supported_lane_count {
     ($($lanes:literal),+) => {
         $(
             impl SupportedLaneCount for LaneCount<$lanes> {
-                type BitMask = [u8; ($lanes + 7) / 8];
-                const EMPTY_BIT_MASK: Self::BitMask = [0; ($lanes + 7) / 8];
+                const LEN: usize = ($lanes as usize).div_ceil(8);
+                type BitMask = [u8; Self::LEN];
+                const EMPTY_BIT_MASK: Self::BitMask = [0; Self::LEN];
                 const FULL_BIT_MASK: Self::BitMask = {
-                    const LEN: usize = ($lanes + 7) / 8;
```

Comment:
Just curious, it didn't work in this scope?

Reply:
I don't think it exists to be used for `BitMask`, but looking at this code again, I'm wondering why I didn't reuse the `LaneCount::BITMASK_LEN`...

Reply:
Now done, seems much better.

Reply:
Thank you!